### ISSUE TITLE
Outgoing call is getting disconnected immediately after making the call

### ISSUE THREAD

**Issue by YasarCse** [twilio/voice-quickstart-ios#502]
**Description**
The outgoing call is getting disconnected immediately after making the call. When I make a call from my VoIP app, It's reporting the call successfully but disconnects at the very next moment.

**Log**
2022-07-05 18:05:13.412626+0530 Appname[383:6421] provider:didActivateAudioSession:
2022-07-05 18:05:13.739390+0530 Appname[383:6421] callDidStartRinging:
2022-07-05 18:05:15.406238+0530 Appname[383:6421] Call failed to connect: Connection error
2022-07-05 18:05:15.406379+0530 Appname[383:6421] performVoiceCall() failed
2022-07-05 18:05:16.422828+0530 Appname[383:6421] provider:didDeactivateAudioSession:

**Twilio Voice Version**: %@ : 6.4.1
**XCode** : 13.4.1
**iOS** : 15.5 iPhone 6S

**Comment by bobiechen-twilio** [comment]
Hi @YasarCse 

Please reach out to Twilio Support with more detail such as trace level SDK logs or additional account information (Account SID, TwiML App SID).

From the log you provided it looks like the call was never successfully connected. You should be able to see detailed error info in the debug level SDK log.

**Comment by vitalii-tym** [comment]
I have the same issue. From the error log it is clear that it is trying to connect to some default url, which is obviously can't be alive:

<img width="972" alt="Screenshot 2022-08-12 at 15 40 06" src="https://user-images.githubusercontent.com/5410835/184355847-1c21f3c9-1620-441e-8172-ae889a184472.png">

I have found that I need to copy a specific url to the TwiML function in my account. Which looks like https://quickstart-3XX8-dev.twil.io/make-call

But I have already spent quite a lot of time trying to figure out where I should paste this url. This is not mentioned on documentation.
